azkaban-web-server/src/web/js/azkaban/view/svg-graph.js

```diff
@@ -90,8 +91,12 @@ azkaban.SvgGraphView = Backbone.View.extend({
     $(this.mainG).empty();
 
     this.graphBounds = this.renderGraph(this.model.get("data"), this.mainG);
-    this.expandAllFlows(null, graphDepth || 1);
-    this.resetPanZoom(0);
+    
+    if (this.model.get("autoExpand")) {
```

Comment:
Should it be this.model.get("autoExpand**Flows**") ? (Based the binding statement of line 36 above).

Reply:
Yes. Naming should be common.